src/scenes/Oracle/index.js

```diff
@@ -235,101 +250,58 @@ class OraclePage extends React.Component {
     this.props.getOracles([
       { topicAddress: this.state.topicAddress },
     ], undefined);
-    // TODO (LIVIA): NEED TO TXS FOR THE ENTIRE TOPIC
+
     this.props.getTransactions([
-      { oracleAddress: this.state.address },
+      { topicAddress: this.state.topicAddress },
```

Comment:
is this dependable on every transaction must hv the topicAddress attribute?

Reply:
yup. only the Oracle and Topic detail pages will we query by `topicAddress`.